### PR TITLE
Update appt ids in docs to be more accurate

### PR DESCRIPTION
**Pull request by jperk51**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Update appt ids in docs to be more accurate

## Related issue(s)
N/A

## Testing done

None. Docs only.

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/mobile/docs/examples/appointments/cc_appointment.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 data:
   - type: appointment
-    id: "23fe358d-6e82-4541-804c-ce7562ba28f4"
+    id: "8a48e8db6d70a38a016d72b354240002"
```
[the remark this answers is not in the document]
Comment (jperk51) [reply modules/mobile/docs/examples/appointments/cc_appointment.yml:3]: I'm honestly not sure. Let me see if I can find an example

[the remark this answers is not in the document]
Comment (jperk51) [reply modules/mobile/docs/examples/appointments/cc_appointment.yml:3]: I also didn't regen the docs so I need to do that still anyway

[the remark this answers is not in the document]
Comment (jperk51) [reply modules/mobile/docs/examples/appointments/cc_appointment.yml]: Found a cassette with an example and it seems like they do have the same style ids so I updated them

**Review by aherzberg — APPROVED** [review]
LGTM

**Pull request merged**